src/twitch.js

```diff
@@ -56,14 +57,12 @@ TwitchCtrl.prototype.getUser = function(username) {
 * @param {Integer} options.offset : object offset for pagination {Default: 0}
 * @returns {Promise.<string, Error>} : resolve JSON data or rejects an error
 */
-TwitchCtrl.prototype.getFeaturedStreams = function(options) {
+Twitch.prototype.getFeaturedStreams = function(options) {
     return new Promise((resolve, reject) => {
         // set our URL for working with the api
         let url = "https://api.twitch.tv/kraken/streams/featured"
         if(options) {
```

Comment:
depending on how we feel, we should be able to remove this `if (options)` check because `query-string` [checks the passed in object here](https://github.com/sindresorhus/query-string/blob/master/index.js#L176)